configs/webpack-config-compass/src/webpack-plugin-start-electron.ts

```diff
@@ -134,4 +190,44 @@ export class WebpackPluginStartElectron {
       await once(childProcess, 'exit');
     }
   }
+
+  private setupKeypressListeners() {
+    if (!process.stdin.isTTY) {
+      return;
+    }
+    process.stdin.setRawMode(true);
+    readline.emitKeypressEvents(process.stdin);
+    process.stdin.on('keypress', (str: string) => {
```

Comment:
@addaleax pinging you just in case I'm doing something really nasty here, not sure if that is a better way of handling keyboard input in Node.js

Reply:
No, exactly the right thing :slightly_smiling_face: You can use the second argument of the event handler if you want a bit more verbosity, but that shouldn’t be very different from just adding comments the way you did here

Reply:
Aha, gotcha! Thanks for giving it a look so quickly 😊